backend/src/org/commcare/suite/model/Detail.java

```diff
@@ -65,6 +65,10 @@
     // Force the activity that is showing this detail to show itself in landscape view only
     private boolean forceLandscapeView;
 
+    // If this detail is being shown in an entity list, set focus to the LAST item in the list
+    // when it is loaded, instead of the first
+    private boolean focusToBottomOfEntityList;
```

Comment:
It's old J2ME Stuff. Will may still be using it, actually, I'd check with
them before killing.

-Clayton

On Mon, Sep 12, 2016 at 1:12 PM, Aliza Stone <notifications@github.com>
wrote:

> In backend/src/org/commcare/suite/model/Detail.java
> <https://github.com/dimagi/commcare-core/pull/396#discussion_r78413501>:
>
> > @@ -65,6 +65,10 @@
> >      // Force the activity that is showing this detail to show itself in landscape view only
> >      private boolean forceLandscapeView;
> >
> > +    // If this detail is being shown in an entity list, set focus to the LAST item in the list
> > +    // when it is loaded, instead of the first
> > +    private boolean focusToBottomOfEntityList;
>
> yea that sounds right. can we kill it then?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-core/pull/396/files/b5348ead9a84110829d5705d3196901b61bc5e3c#r78413501>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJdupuY9qukNU06DFDC8tRheoyUHSjXks5qpYgagaJpZM4J6tW1>
> .
>